GooglePlayInstant/InstallLauncher.cs

```diff
@@ -41,20 +40,40 @@ namespace GooglePlayInstant
     /// <a href="https://developers.google.com/android/reference/com/google/android/gms/instantapps/InstantApps">
     /// Google Play Services' InstantApps class</a>.
     /// </summary>
-    public static class InstallLauncher
-    {
+    public static class InstallLauncher {
         private const string IntentActionInstantAppInstall = "com.google.android.finsky.action.IA_INSTALL";
         private const int IgnoredRequestCode = 1001;
 
         /// <summary>
         /// Shows a dialog that allows the user to install the current instant app.
         /// </summary>
-        public static void ShowInstallPrompt()
-        {
-            using (var activity = UnityPlayerHelper.GetCurrentActivity())
-            using (var postInstallIntent = CreatePostInstallIntent(activity))
-            {
-                ShowInstallPrompt(activity, IgnoredRequestCode, postInstallIntent, null);
+        public static void ShowInstallPrompt () {
+            using (var activity = UnityPlayerHelper.GetCurrentActivity ())
+            using (var postInstallIntent = CreatePostInstallIntent (activity)) {
+                ShowInstallPrompt (activity, IgnoredRequestCode, postInstallIntent, null);
+            }
+        }
+
+        public static void ShowInstallPrompt (string referrerId) {
```

Comment:
I assume this method was created to allow passing a referredId only. Please see my suggestion above and delete this if it seems OK.

Reply:
fixed per your recommendation